prow/plugins/assign/assign.go

```diff
@@ -140,6 +146,13 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	if len(assign) > 0 {
 		log.Printf("Adding assignees to %s/%s#%d: %v", ae.org, ae.repo, ae.number, assign)
 		if err := gc.AssignIssue(ae.org, ae.repo, ae.number, assign); err != nil {
+			if mu, ok := err.(*github.MissingUsers); ok {
```

Comment:
We don't want to propagate up the error in this case. `MissingUsers` isn't a problem with prow, it's a problem with what the user tried to tell prow to do.